packages/contracts-bedrock/interfaces/L1/IL1CrossDomainMessenger.sol

```diff
@@ -3,18 +3,18 @@ pragma solidity ^0.8.0;
 
 import { ICrossDomainMessenger } from "interfaces/universal/ICrossDomainMessenger.sol";
 import { ISuperchainConfig } from "interfaces/L1/ISuperchainConfig.sol";
-import { IOptimismPortal } from "interfaces/L1/IOptimismPortal.sol";
+import { IOptimismPortal2 } from "interfaces/L1/IOptimismPortal2.sol";
 import { ISystemConfig } from "interfaces/L1/ISystemConfig.sol";
 
 interface IL1CrossDomainMessenger is ICrossDomainMessenger {
-    function PORTAL() external view returns (IOptimismPortal);
+    function PORTAL() external view returns (IOptimismPortal2);
     function initialize(
         ISuperchainConfig _superchainConfig,
-        IOptimismPortal _portal,
+        IOptimismPortal2 _portal,
         ISystemConfig _systemConfig
     )
         external;
-    function portal() external view returns (IOptimismPortal);
+    function portal() external view returns (IOptimismPortal2);
```

Comment:
Now that I see the PR, I think we should not modify the interface by changing this type. 

I know that you raised this previously, so I apologize for the rework.